### ISSUE TITLE
Support for Groups

### ISSUE THREAD

**Issue by paragbatavia** [jlvaillant/intellicenter#26]
Currently - circuits that are configured as groups do not show up. This seems to be used for water features in a pool (i.e., waterfalls), where multiple pump states need to be set simultaneously.

**Comment by jlvaillant** [comment]
They should appear in HA as switches but ONLY if they are tagged as features on the IntelliCenter panel (and appear under the features page on the IntelliCenter mobile app).

**Comment by paragbatavia** [comment]
Got it – it only comes up under groups, so I likely need to change something in the intellicenter panel. 

 

Thanks much – and thanks for developing this!

**Comment by thewolfman56** [comment]
I just turned on my Intellicenter a week ago and after using the intellicenter app for a day, I installed this integration with HACS.  My Circuit Group is "Water Slide".  It turns my valve 1 (water feature) to on and then changes my valve 2 (Slide) to on.  This Circuit Group shows up under "Features" in my intellicenter app, but when I check the Intellicenter Control Panel , there is no box to check "Features" as there are for individual circuits.

I would love to be able to see this Circuit Group in my Home Assistant, as the lights respond much quicker than the pentair app.  I can also turn on my waterslide through Home Assistant as well by individually turning on the individual valves, but I would like the Circuit Group so I can make it easier for my wife's dashboard.
